haros_config.yaml

```diff
@@ -3,4 +3,8 @@
 packages:
```

Comment:
Why did this file already existed in develop? was it only to test the CI?

Reply:
Yeah i comitted this file by mistake in my last PR. Sowwyyyyy :man_shrugging: